src/components/ViewQuiz.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+
+import { Card, ListGroup } from 'react-bootstrap';
+import { connect } from 'react-redux';
+
+import ViewQuestions from './ViewQuestions';
+
+class ViewQuiz extends Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    const { id } = this.props.match.params;
+    const { quizzes, questions } = this.props;
+    const quiz = quizzes.find(quiz => quiz.id == id);
+    const { title, description, questions: questionIds } = quiz;
```

Comment:
Why didn't you use functional component, you already have all your logic inside a render method

Reply:
fixed